internal/kube/controller/config.go

```diff
@@ -47,6 +47,6 @@ func BoundConfig(flags *flag.FlagSet) (*Config, error) {
 	iflag.StringVar(flags, &c.Kubeconfig, "kubeconfig", "KUBECONFIG", "", "A path to the kubeconfig file to use")
 	iflag.StringVar(flags, &c.WatchNamespace, "watch-namespace", "WATCH_NAMESPACE", metav1.NamespaceAll, "The Kubernetes namespace the controller should monitor for controlled resources (will monitor all if not specified)")
 	iflag.StringVar(flags, &c.Name, "name", "CONTROLLER_NAME", "", "A name identifying the controller. If not specified it will be deduced from the hostname.")
-	iflag.BoolVar(flags, &c.RequireAnnotation, "sites-require-annotation", "SITES_REQUIRE_ANNOTATION", false, "If set, this controller instance will only process sites with an annotation matching the controller's namespace qualified name")
+	iflag.BoolVar(flags, &c.RequireExplicitControl, "require-explicit-control", "REQUIRE_EXPLICIT_CONTROL", false, "If set, this controller instance will only process resources in which there is a ConfigMap named skupper with an entry 'controller' whose value matches the controller's namespace qualified name. Controllers watching a single namespace require that ConfigMap regardless of this setting.")
```

Comment:
`REQUIRE_EXPLICIT_CONTROL` needs to be added into `config/overlays/namespace/patch-add-env-var-watch-namespace.yaml`

Reply:
> `REQUIRE_EXPLICIT_CONTROL` needs to be added into `config/overlays/namespace/patch-add-env-var-watch-namespace.yaml`

That option does not apply to namespaces scoped controllers as they always require explicit control. What I did add was a configmap to grant that control in the namespaces scoped case.

The option there is only required where you want to run multiple cluster scoped controllers (which at present I am assuming is a fringe use case)